.changeset/rare-actors-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/virtualizer': major
+---
+
+Change window to globalThis in Virtualizer.ts to support SSR scenarios.
```

Comment:
I suppose this is technically a breaking change warranting a major version. Feels a bit weird to do a major bump just for this but if we're strictly following semver... I would also remove the wording that this "supports" SSR scenarios.

```suggestion
BREAKING: Replace top-level access to `window` with `globalThis` so that loading the module won't error in non-browser environments.

A polyfill for `globalThis` may be needed if you are targeting an environment that does not support it.
```